src/curriculum/level1/learningLetters.js

```diff
@@ -264,6 +264,53 @@ const lesson7 = {
   ],
 };
 
+const lesson8 = {
+  letters: {
+    ⲥ: getLetterExplanation('ⲥ'),
+    ⲧ: getLetterExplanation('ⲧ'),
+    Ϫ: getLetterExplanation('Ϫ'),
+  },
+  exampleWords: [
+    {word: 'Ϩⲓϫⲉⲛ', pronunciation: 'aghapee'},
+    {word: 'Ⲡⲗⲁⲥⲙⲁ', pronunciation: 'psyshee'},
+    {word: 'Ⲉⲛⲧⲟⲗⲏ', pronunciation: 'qeemi'},
+    {word: 'Ⲥⲙⲟⲩ', pronunciation: 'kyryos'},
+    {word: 'Ϫⲱⲣ', pronunciation: 'sheroobim'},
+    {word: 'Ⲉⲛⲧⲏⲙⲁ', pronunciation: 'sheroobim'},
+    {word: 'Ⲕⲟⲥⲙⲟⲥ', pronunciation: 'sheroobim'},
+    {word: 'Ⲥⲙⲏ', pronunciation: 'sheroobim'},
+  ],
+  vocab: [
+    {word: 'Ⲥ̀ⲙⲟⲩ', translation: 'Bless'},
+    {word: 'Ⲕⲟⲥⲙⲟⲥ', translation: 'World'},
+    {word: 'Ⲕ̀ⲥ̀ⲙⲁⲣⲱⲟⲩⲧ', translation: 'Blessed Are Youo'},
+    {word: 'Ϫⲟⲙ', translation: 'Power'},
+    {word: 'Ϫⲱⲙ', translation: 'Book'},
+  ],
+  practiceWords: [
+    {word: 'Ⲕⲟⲩϫⲓ', pronunciation: 'kouji'},
+    {word: 'Ϫⲁϫⲓ', pronunciation: 'gaji'},
+    {word: 'Ⲡⲓⲣⲁⲥⲙⲟⲥ', pronunciation: 'pirazmos'},
```

Comment:
Pr punctuation should be pirasmos

Reply:
So i looked it up, the word is Greek (esp that it ends with ⲟⲥ)
Following the rule of Ⲥ followed by Ⲙ in Greek words, it's pronounced as Z. Such as Kozmos for ⲕⲟⲥⲙⲟⲥ.

Reply:
I stand corrected. I've never seen that word before.

Reply:
I reached out to a Coptic Scholar and it seems there some ambiguity here.
I asked... 
>Ⲡⲓⲣⲁⲥⲙⲟⲥ
>Is this pronounced pirasmos or pirazmos?

He replied...
>Birasmos with an “s” sound
>There isn’t any universal rule for all words. The pronunciation of the letter is dependent on the use of the word when spoken, not on how it is written.
>BTW - all of these new rules for pronunciation of words only began in the 19th century when the pronunciation of Coptic was changed based on modern Greek values for the letters. Even then, the new system didn’t catch on reply until the early 20th century.

Reply:
true. and as this is the accepted pronunciation for hymns, the goal of the app is to go with the currently accepted rules (which could in fact be different than the historic or academic pronunciation).